src/js/modules/infragistics.ui.editors.js

```diff
@@ -1813,6 +1796,9 @@
 			this._superApply(arguments);
 			this._changeLocaleForNotifier();
 		},
+		changeRegional: function() {

```

Comment:
@Lipata This should probably be in numeric (since text is not affected by regional changes)?
Also should it be unconditional displayValue? What about edit mode?